builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -118,3 +157,73 @@ func resourceAwsS3BucketDelete(d *schema.ResourceData, meta interface{}) error {
 	}
 	return nil
 }
+
+func updateWebsite(s3conn *s3.S3, d *schema.ResourceData) error {
+	website := d.Get("website").(bool)
+	bucket := d.Get("bucket").(string)
+	indexDocument := d.Get("index_document").(string)
+	errorDocument := d.Get("error_document").(string)
+
+	if website {
+		websiteConfiguration := &s3.WebsiteConfiguration{}
+
+		if indexDocument != "" {
+			websiteConfiguration.IndexDocument = &s3.IndexDocument{Suffix: aws.String(indexDocument)}
+		}
+
+		if errorDocument != "" {
+			websiteConfiguration.ErrorDocument = &s3.ErrorDocument{Key: aws.String(errorDocument)}
+		}
+
+		putInput := &s3.PutBucketWebsiteInput{
+			Bucket:               aws.String(bucket),
+			WebsiteConfiguration: websiteConfiguration,
+		}
+
+		log.Printf("[DEBUG] S3 put bucket website: %s", putInput)
+
+		_, err := s3conn.PutBucketWebsite(putInput)
+		if err != nil {
+			return fmt.Errorf("Error putting S3 website: %s", err)
+		}
+	} else {
+		deleteInput := &s3.DeleteBucketWebsiteInput{Bucket: aws.String(bucket)}
+
+		log.Printf("[DEBUG] S3 delete bucket website: %s", deleteInput)
```

Comment:
I'm not sure what this should output here, something about the website? Or the bucket name? Right now for me it outputs `[DEBUG] S3 delete bucket website: &{%!s(*string=0xc20834f270) {%!s(bool=false)}}`